### DISCUSSION TITLE
New entity IDs in latest version

### DISCUSSION THREAD

**Discussion by LeiChat** [signalkraft/mypyllant-component#68] [Q&A]
Is there a way to amend existing entities before installing the update so that the history / long-term statistics are retained, please?

**Comment by ml1nk** [comment]
In theory you should be able to manually merge the old entities with the new entities in the database directly. I'm sorry, but there isn't a easy way that i know to do this. Writing a automated migration is to complicated and error prone, at least with my little understanding of HACS, but If you want to try i will help.

**Reply by LeiChat** [reply]
Appreciate the info, thank you.
Fingers crossed the Nabu Casa team introduce a way to merge them more simply in the future.
